### PR TITLE
Fixing mongodb and hygieia api configuration

### PR DESCRIPTION
**Pull request by jdewinne**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,21 +1,24 @@
 mongodb:
-  image: mongo:latest
-  container_name: mongodb
-  command: mongod --smallfiles
-  ports:
-   - "27017:27017"
-  volumes:
-   - ./mongo:/data/db:rw
-  volume_driver: local
+  image: bitnami/mongodb:latest
```
Comment (ebosveld) [inline docker-compose.yml]: Why did you change this to a Bitnami image over the official MongoDB image? What is the value of that change?

Comment (jdewinne) [inline docker-compose.yml]: Sorry for the late reply. I think it had to do with https://github.com/docker-library/mongo/issues/158
But that might now be fixed on the monmgodb side.

**Comment by satishc1** [comment]
@tabladrum @jdewinne Replaced the bitnami with the standard image.
